src/uwtools/tests/test_cli.py

```diff
@@ -192,17 +192,17 @@ def test__dispatch_config(params):
     submode, funcname = params
     args = ns()
     vars(args).update({STR.submode: submode})
-    with patch.object(cli, funcname) as m:
+    with patch.object(cli, funcname) as module:
```

Comment:
`module` is probably not a good name here, as the object being patched is not a module, but rather a function. I think I meant the `m` to mean `mock`, but obviously that was also a bad name since it led to this confusion!

`func` might be a good name here, as that's an idiomatic Python name for a variable that refers to a function object.

The same comment applies similarly elsewhere in this file, though `func` will not be appropriate in all cases.